app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,65 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user,   only: [:edit, :update]
```

Comment:
```suggestion
  before_action :get_user,   only: %i[edit update]
```
こういった場合も%記法を用いることが多いです！